Cargo.toml

```diff
@@ -23,6 +23,7 @@ members = [
   "src/tools/rustfmt",
   "src/tools/miri",
   "src/tools/rustdoc-themes",
+  "src/rustc_lexer",
```

Comment:
I'm not sure why is the crate added to this list (other crates are not) and what are the built system consequences.

Reply:
Yeah it shouldn't need to be here, if other crates depend on it.